rss2email/util.py

```diff
@@ -105,7 +105,7 @@ def import_name(obj):
       ...
     ValueError: abc
     """
-    pickle = _pickle.dumps(obj)
+    pickle = _pickle.dumps(obj, protocol=3)
```

Comment:
```suggestion
    name = f'{obj.__module__} {obj.__qualname__}'
    if import_function(name) is obj:
        return name
    raise ValueError(obj)
```

Reply:
Ah, now I see where you were going with this. We should use classic format-strings though, as f-strings are Python >= 3.6 and this project only requires 3.4.

Reply:
Yeah, please don’t use all new python features just because they exist :)